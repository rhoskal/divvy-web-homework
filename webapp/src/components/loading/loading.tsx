import React from "react";

/**
 * Component
 */

export function Loading(): JSX.Element {
  return (
    <div>
      <span>Loading...</span>
    </div>
  );
}
