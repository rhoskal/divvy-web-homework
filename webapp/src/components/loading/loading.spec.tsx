import React from "react";
import { render } from "@testing-library/react";

import { Loading } from "./loading";

describe("[Component] Loading", () => {
  it("should match snapshot", () => {
    const { container } = render(<Loading />);

    expect(container.firstChild).toMatchSnapshot();
  });
});
