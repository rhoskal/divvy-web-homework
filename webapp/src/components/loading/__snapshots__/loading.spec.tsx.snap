// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[Component] Loading should match snapshot 1`] = `
<div>
  <span>
    Loading...
  </span>
</div>
`;
